### ISSUE TITLE
Running inside Gnome crashes Gnome

### ISSUE THREAD

**Issue by Jacalz** [fyne-io/fynedesk#189]
Trying to run `fynedesk` inside Gnome causes Gnome to have some sort of error and crash. This happens both when just running `./fynedesk` and when running the tests. It is very annoying because it makes it impossible to run the tests locally.

**Comment by andydotxyz** [comment]
Are you able to capture any logging from Gnome?

**Comment by Jacalz** [comment]
Yeah sure. I found it the forts time, but the log seemed to be gone when it crashed the second time. Will try again tomorrow.

**Comment by Jacalz** [comment]
These are the logs found in the `Problem Reporting` application found in Fedora. I could not manage to find how to export it to a single log file so this is unfortunately data split out into multiple files. I hope it can be of use either way.
[fynedesk-crashes-gnome.zip](https://github.com/fyne-io/fynedesk/files/6564650/fynedesk-crashes-gnome.zip)
